src/java.base/share/classes/sun/nio/ch/FileChannelImpl.java

```diff
@@ -537,106 +547,206 @@ public void force(boolean metaData) throws IOException {
     // Assume at first that the underlying kernel supports sendfile();
```

Comment:
It's not just sendfile() any more so maybe this comment should be broadened.

Reply:
This PR doesn't change which syscalls are used but I agree that this comment is a bit outdated, we can make it sendfile/equivalent to avoid enumerating the different syscalls and implementations that can be used.

Reply:
Sounds good.